compiler/src/dotty/tools/dotc/interactive/Completion.scala

```diff
@@ -207,11 +209,17 @@ object Completion {
      * considered.
      */
     def addMemberCompletions(qual: Tree)(implicit ctx: Context): Unit = {
-      addAccessibleMembers(qual.tpe)
-      if (!mode.is(Mode.Import)) {
-        // Implicit conversions do not kick in when importing
-        implicitConversionTargets(qual)(ctx.fresh.setExploreTyperState())
-          .foreach(addAccessibleMembers)
+      if (qual.tpe =:= defn.NothingType)
+        ()
+      else if (qual.tpe =:= defn.NullType)
```

Comment:
I don't understand, what other methods than those on AnyRef does completion find on null ? Where do they come from ?

Reply:
Instead if `=:=` it's more idiomatic to use `isRef`. I.e. `qual.tpe.isRef(defn.NothingClass)`. (It's also faster).

Reply:
Done

Reply:
There's `isBottomType`

Reply:
Used `isBottomType`